pkg/catalog/helm/setup.go

```diff
@@ -43,7 +46,11 @@ func NewNoUpdate(catalog *v3.Catalog) (*Helm, error) {
 	if catalog == nil || catalog.Name == "" {
 		return nil, errors.New("Catalog is not defined for helm")
 	}
-	return newCache(catalog), nil
+	cachedCatalog, err := newCache(catalog)
```

Comment:
Just return this call, it's the same thing

Reply:
fixed